src/ui/Island/RealmDetails/StakingForms/StakeForm.tsx

```diff
@@ -137,7 +136,7 @@ export default function StakeForm({ isDisabled }: { isDisabled: boolean }) {
           disabled={
             isDisabled ||
             !isStakeAmountValid ||
-            !isValidInputAmount(stakeAmount)
+            !isValidInputAmount(`${stakeAmount}`)
```

Comment:
`isValidInputAmount` expects it's argument to be a float not a stringified bigint, I would check if it won't give us some weird side effects